README.md

```diff
@@ -93,6 +85,24 @@ You can not yet retrieve automatically all similar sequences with the noise redu
 
 CPT has 3 meta parameters that need to be tuned. You can check how to tune them in the [documentation][1]. To tune you can use the `model_selection` module from `sklearn`, you can find an example [here][3] on how to.
 
+## Benchmark
+
+The benchmark has been made on the FIFA dataset, the data can be found on the [SPMF website][4].
+
+Using multithreading, `CPT` was able to perform around 5000 predictions per second.
+
+Without multithreading, `CPT` predicted around 1650 sequences per second.
+
+Details on the benchmark can be found [here](benchmark).
+
+## Further reading
+
+A study has been made on how to reduce dataset size, and so training / testing time using PageRank on the dataset.
+
+The study has been published in IJIKM review [here][5]. An overall performance of 10-40% has been observed with this technique on the prediction time without any accuracy loss.
```

Comment:
"performance of 10-40% has been observed"
Did you mean a performance improvement?